app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  has_many :posts
```

Comment:
- [ ] Here you can add `foreign_key: 'user_id'` after comments or likes.

`has_many :comments, foreign_key: 'author_id'` like this.